database/sqlite/sqlite_health.c

```diff
@@ -0,0 +1,1367 @@
+            buffer_sprintf(wb, ",\n\t\t\"to\": \"%s\"", sqlite3_column_text(res_alert, 21));
+        }
+        if (sqlite3_column_type(res_alert, 22) != SQLITE_NULL) {
+            buffer_sprintf(wb, ",\n\t\t\"info\": \"%s\"", sqlite3_column_text(res_alert, 22));
+        }
+        if (sqlite3_column_type(res_alert, 23) != SQLITE_NULL) {
+            buffer_sprintf(wb, ",\n\t\t\"delay\": \"%s\"", sqlite3_column_text(res_alert, 23));
+        }
+        if (sqlite3_column_type(res_alert, 24) != SQLITE_NULL) {
+            buffer_sprintf(wb, ",\n\t\t\"options\": \"%s\"", sqlite3_column_text(res_alert, 24));
+        }
+        if (sqlite3_column_type(res_alert, 25) != SQLITE_NULL) {
+            buffer_sprintf(wb, ",\n\t\t\"repeat\": \"%s\"", sqlite3_column_text(res_alert, 25));
+        }
+        if (sqlite3_column_type(res_alert, 26) != SQLITE_NULL) {
+            buffer_sprintf(wb, ",\n\t\t\"host_labels\": \"%s\"", sqlite3_column_text(res_alert, 26));
+        }
+        if (sqlite3_column_type(res_alert, 27) != SQLITE_NULL) {
+            buffer_sprintf(wb, ",\n\t\t\"p_db_lookup_dimensions\": \"%s\"", sqlite3_column_text(res_alert, 27));
+        }
+        if (sqlite3_column_type(res_alert, 28) != SQLITE_NULL) {
+            buffer_sprintf(wb, ",\n\t\t\"p_db_lookup_method\": \"%s\"", sqlite3_column_text(res_alert, 28));
+        }
+        if (sqlite3_column_type(res_alert, 29) != SQLITE_NULL) {
+            buffer_sprintf(wb, ",\n\t\t\"p_db_lookup_options\": \"");
+            buffer_data_options2string(wb, sqlite3_column_int(res_alert, 29));
+            buffer_strcat(wb, "\"");
+        }
+        if (sqlite3_column_type(res_alert, 30) != SQLITE_NULL) {
+            buffer_sprintf(wb, ",\n\t\t\"p_db_lookup_after\": %d", sqlite3_column_int(res_alert, 30));
+        }
+        if (sqlite3_column_type(res_alert, 31) != SQLITE_NULL) {
+            buffer_sprintf(wb, ",\n\t\t\"p_db_lookup_before\": %d", sqlite3_column_int(res_alert, 31));
+        }
+        if (sqlite3_column_type(res_alert, 32) != SQLITE_NULL) {
+            buffer_sprintf(wb, ",\n\t\t\"p_update_every\": %d", sqlite3_column_int(res_alert, 32));
+        }
+
+        if (unlikely(rc != SQLITE_OK))
+            error_report("Failed to reset the prepared statement when reading chart config with hash");
+        buffer_strcat(wb, "\n\t}");
+    }
+
+    buffer_sprintf(wb, "\n]");
+
+    failed:
+    rc = sqlite3_finalize(res_alert);
+    if (unlikely(rc != SQLITE_OK))
+        error_report("Failed to finalize the prepared statement when reading chart config with hash");
+
+    return;
+}
+
+int alert_hash_and_store_config(
+    uuid_t hash_id,
+    struct alert_config *cfg)
+{
+    EVP_MD_CTX *evpctx;
+    unsigned char hash_value[EVP_MAX_MD_SIZE];
+    unsigned int hash_len;
+    evpctx = EVP_MD_CTX_create();
+    EVP_DigestInit_ex(evpctx, EVP_sha256(), NULL);
+
+    error("HASH asked to hash [%s]", cfg->alarm ? cfg->alarm : cfg->template_key);
```

Comment:
We should probably remove this or report with a suitable debug flag

Reply:
Ah, removed, was left there for some tests....